pkg/app/app.go

```diff
@@ -164,6 +165,29 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval := a.config.Broker.RefreshInterval
+	if interval != 0 {
+		ticker := time.NewTicker(time.Duration(interval) * time.Second)
```

Comment:
time.Seconds is type time.Duration. If I don't type cast it as time.Duration I get an error.